pkgs/tools/system/htop/default.nix

```diff
@@ -1,19 +1,23 @@
 { lib, fetchurl, stdenv, ncurses,
-IOKit }:
+IOKit, python }:
 
 stdenv.mkDerivation rec {
   name = "htop-${version}";
-  version = "2.0.2";
+  version = "2.1.0";
 
   src = fetchurl {
-    sha256 = "11zlwadm6dpkrlfvf3z3xll26yyffa7qrxd1w72y1kl0rgffk6qp";
     url = "http://hisham.hm/htop/releases/${version}/${name}.tar.gz";
+    sha256 = "0j07z0xm2gj1vzvbgh4323k4db9mr7drd7gw95mmpqi61ncvwq1j";
   };
 
   buildInputs =
-    [ ncurses ] ++
+    [ ncurses python ] ++
```

Comment:
Isn't it sufficient to put this in `nativeBuildInputs`?

Reply:
Indeed 👍
It's only used for build time code generation